### DISCUSSION TITLE
Question about Appdata folder for Atom

### DISCUSSION THREAD

**Discussion by rczink01** [community/community#27614] [Other Feature Feedback, Questions, & Ideas]
Hi, all.

Forgive me if this has been discussed elsewhere, but I have been using Atom, and I notice it creates a large amount of data within my C:\users\first.last\AppData folder. Is there an easy way to purge files from this folder to free up some space?

Richard

**Comment by FahdSeddik** [comment]
Hey! @rczink01 
I think this [link](https://theitbros.com/appdata-folder/) might help.

**Comment by MrSarno** [comment]
Hi @rczink01,

I'm not sure if you've heard, however the team behind Atom [have decided](https://github.blog/2022-06-08-sunsetting-atom/) to stop working on it. For that reason alone, I would suggest you consider looking at alternative editors. Free options include [VS Code](https://code.visualstudio.com/) and [IntelliJ (Community)](https://www.jetbrains.com/idea/download/). [Sublime](https://www.sublimetext.com/), a paid option, is also excellent.

Naturally, if you're no longer using Atom, you can uninstall the program & reclaim that space.
